object-api-multipart_test.go

```diff
@@ -1725,7 +1725,7 @@ func testObjectCompleteMultipartUpload(obj ObjectLayer, instanceType string, t *
 
 	uploadIDs = append(uploadIDs, uploadID)
 	// Parts with size greater than 5 MB.
-	// Generating a 6MB byte array.
+	// Generating a 16MB byte array.
 	validPart := bytes.Repeat([]byte("0123456789abcdef"), 1024*1024)
```

Comment:
Is there a particular reason to make this 16MB? if they are valid parts why not just make it like 6MB or 8MB, 16MB for each part is not necessary for these tests.